assets/src/dashboard/theme.js

```diff
@@ -99,6 +99,13 @@ const theme = {
       lineHeight: '20px',
       weight: '500',
     },
+    pill: {
+      family: "'Google Sans', 'Sans Serif'",
```

Comment:
`"'Google Sans', sans-serif"`

Reply:
thanks, the structure of this theme through me off. should be fixed.